articles/case_studies/fec-javascript-testing.md

```diff
@@ -0,0 +1,70 @@
+# Testing and refactoring JavaScript with FEC
+
+Until August of this year, the JavaScript for the OpenFEC project had no unit tests. This wasn't because the development team wasn't interested in testing--our backend code was already thoroughly tested. Instead, our JavaScript code just wasn't pleasant to test. Here's an example of the coding style that prevailed not long ago:
```

Comment:
Agreed, will revise to be explicit that we're not talking about node. I wrote this in stackedit and didn't bother with intra-paragraph line breaks, but I can add them if it's getting in the way of review.